src/main/java/net/datafaker/providers/base/Locality.java

```diff
@@ -102,8 +102,8 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
         for (String s: paths) {
             try {
                 Files.walkFileTree(Paths.get(s).toAbsolutePath(), visitor);
-            } catch (Exception e) {
-                throw new RuntimeException(e);
+            } catch (IOException e) {
+                throw new RuntimeException("Failed to read path " + s, e);
```

Comment:
```suggestion
                throw new RuntimeException("Failed to read path '" + s + "'", e);
```
sometimes when string is empty such output with quotes could save time during debug
another version with formatting is also ok

Reply:
Good point, fixed.